models/mac_interface_map.go

```diff
@@ -53,11 +54,34 @@ type MacInterfaceMapItems0 struct {
 	LogicalNicName string `json:"logical_nic_name,omitempty"`
 
 	// mac address present on the host
+	// Pattern: ^([0-9A-Fa-f]{2}[:]){5}([0-9A-Fa-f]{2})$
 	MacAddress string `json:"mac_address,omitempty"`
 }
 
 // Validate validates this mac interface map items0
 func (m *MacInterfaceMapItems0) Validate(formats strfmt.Registry) error {
+	var res []error
+
+	if err := m.validateMacAddress(formats); err != nil {
+		res = append(res, err)
+	}
+
+	if len(res) > 0 {
+		return errors.CompositeValidationError(res...)
+	}
+	return nil
+}
+
+func (m *MacInterfaceMapItems0) validateMacAddress(formats strfmt.Registry) error {
+
+	if swag.IsZero(m.MacAddress) { // not required
+		return nil
+	}
+
+	if err := validate.Pattern("mac_address", "body", string(m.MacAddress), `^([0-9A-Fa-f]{2}[:]){5}([0-9A-Fa-f]{2})$`); err != nil {
```

Comment:
nit: the regex pattern could be defined as `const`

Reply:
This file is generated from the swagger.yaml file - see the first line of it:
`// Code generated by go-swagger; DO NOT EDIT.`

Reply:
this is a swagger generated file